packages/legacy-client/src/client/objectSets/OsdkObjectSet.test.ts

```diff
@@ -243,12 +245,12 @@ describe("OsdkObjectSet", () => {
         pageToken: "fakePageToken",
       }),
     );
-    expect(result.type).toEqual("ok");
+    const selectedObjects = unwrapResultOrThrow(result);
```

Comment:
Its kind of sad that `selectedObjects` is unused and also doesn't have a `_` prefix

Reply:
Was using that to be able to see what the error was quickly, but probably should have a eslint rule for no-unused-vars